storage/database.go

```diff
@@ -215,6 +218,30 @@ func (d *db) Options() Options {
 	return d.opts
 }
 
+func (d *db) AssignShardSet(shardSet sharding.ShardSet) {
+	d.RLock()
+	defer d.RUnlock()
+	for _, ns := range d.namespaces {
+		ns.AssignShardSet(shardSet)
+	}
+	if d.bootstraps > 0 {
```

Comment:
It's to determine if bootstrapping or bootstrapped.  Hence did not want to create a bool called "bootstrapped" because it's really more like "beganInitialBootstrap" which is just confusing.

The int may come in handy later anyhow.